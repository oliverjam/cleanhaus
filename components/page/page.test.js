import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Page from "./page";

jest.unmock("./page");

describe("Page component", () => {
  it("Renders correctly", () => {
    const page = renderer.create(<Page>Lorem Ipsum</Page>).toJSON();
    expect(page).toMatchSnapshot();
  });
});
