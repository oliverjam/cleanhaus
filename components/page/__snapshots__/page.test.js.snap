// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page component Renders correctly 1`] = `
.c0 {
  width: 100%;
  max-width: 40em;
  margin: 0 auto;
  padding: 1em;
}

<main
  className="c0"
>
  Lorem Ipsum
</main>
`;
